tests/access/OwnableBaseSuite.py

```diff
@@ -0,0 +1,67 @@
+import pytest
```

Comment:
Should we separate BaseSuites from test files? I think a subfolder inside `access` like `base_suites`, containing BaseSuites, would allow keeping the focus on test files while traversing the tests. what do you think?

Reply:
If there is going to be only one BaseSuite per folder, then this is probably not required, but then if we find ourselves for some reason with two BaseSuites in one folder, we should refactor that, I think (thinking out loud about organization).

Reply:
> Should we separate BaseSuites from test files? I think a subfolder inside `access` like `base_suites`, containing BaseSuites, would allow keeping the focus on test files while traversing the tests. what do you think?

I was thinking having the `BaseSuite` among the test files is enough to keep it organized. I agree, if there were to be multiple, it would make sense to create a subfolder.

Actually, if we wanted more than one `BaseSuite`, we could just include another class within the `BaseSuite` module. I was considering something like having ERC20Base, ERC20MintableBase, ERC20BurnableBase, etc. and leverage inheritance even further if mint and burn are used in future presets.

Ultimately, if we think it's better for organization, I'm happy to throw the `BaseSuite`s into a subfolder